README.md

```diff
@@ -1 +1,176 @@
-# my-portfolio-website
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>Lincoln's Portfolio</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 Lincoln's Portfolio ](#-lincoln-portfolio-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Lincoln's Portfolio <a name="about-project"></a>
+
+This is my first original project that designed to showcase my catalog, it is build using html, css and linters.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML/">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Added index.html file
+- Added style.css file
+- Added linters
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- Coming soon
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+- A web browser
+- A code editor
+- A terminal
+- Git
+- Node.js
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+run `git clone https://github.com/lincoln1883/hello-microverse.git`
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd my-folder
```

Comment:
- [ ] Kindly make sure you use the appropriate information in each section to make your documentation be more professional. As your project is named `my-portfolio-website` you should update the cloning `link`  and the accessing cloned folder command. i.e.
```
 ...
  cd my-deisired-folder
 git clone https://github.com/lincoln1883/my-portfolio-website.git


  ...

  cd   my-portfolio-website

```